css/project-list-pages.styl

```diff
@@ -43,6 +43,10 @@
     .pill-button
       margin-right: 10px
       padding: 1em
+      color: rgb(64,64,64)
+      font-weight: normal
+      border-color: rgb(64,64,64) !important
```

Comment:
wow - good spot! it picked up so much of the styling it didn't occur to me it might have been the wrong element type. OK I will remove all attempts to modify button styling from this PR. Will you take care of the nested nav at the bottom as well, or shall I do that?